assets/src/apps/authoring/components/Flowchart/rules/create-multiple-choice-rules.ts

```diff
@@ -74,7 +74,7 @@ export const generateMultipleChoiceRules = (
 
   const commonErrorConditionsFeedback: IConditionWithFeedback[] = commonErrorPaths.map((path) => ({
     conditions: createMCQCommonErrorCondition(path, question),
-    feedback: commonErrorFeedback[path.selectedOption - 1] || DEFAULT_CORRECT_FEEDBACK,
+    feedback: commonErrorFeedback[path.selectedOption - 1] || DEFAULT_INCORRECT_FEEDBACK,
```

Comment:
The issue was with the default text that was getting added when an incorrect MCQ option was selected as option when adding new rule.